include/sound/sof.h

```diff
@@ -17,6 +17,8 @@
 
 struct snd_sof_dsp_ops;
 
+struct sof_vfe;
```

Comment:
fix the commit message.

The first sentence mentions 'a Virtio driver" and 'SOF vhost driver". using 'this driver' in following sentences makes it impossible to know which of the two you are referring to.

Also there has been no explanation of what the 'vhost driver' is, so it's still puzzling even with proper grammar...